src/App.tsx

```diff
@@ -41,6 +42,7 @@ const App: FunctionComponent<AppProps> = props => {
 					{props.location.pathname !== APP_PATH.LOGIN_AVO && <Navigation {...props} />}
 					{renderRoutes()}
 					{props.location.pathname !== APP_PATH.LOGIN_AVO && <Footer {...props} />}
+					<Zendesk zendeskKey="2aae0d3b-eb63-48ee-89ef-a7adcfacc410" />
```

Comment:
Als dit ook iets is dat met env variables gezet moet worden, nog een TODO toevoegen.
If not ignore remark 😄

Reply:
ja eigenlijk is dat wel mooier
fixed